netmap/types.proto

```diff
@@ -228,11 +228,31 @@ message NodeInfo {
   State state = 4 [json_name = "state"];
 }
 
+// NeoFS network configuration
+message NetworkConfig {
+  // Single configuration parameter.
+  message Parameter {
+    // Parameter key. Hex-encoded string.
+    bytes key = 1 [json_name = "key"];
+
+    // Parameter value.
+    bytes value = 2 [json_name = "value"];
+  }
+  // List of parameter values.
+  repeated Parameter parameters = 1 [json_name = "parameters"];
+}
+
 // Information about NeoFS network
 message NetworkInfo {
     // Number of the current epoch in the NeoFS network.
     uint64 current_epoch = 1 [json_name = "currentEpoch"];
 
     // Magic number of the sidechain of the NeoFS network.
     uint64 magic_number = 2 [json_name = "magicNumber"];
+
+    // MillisecondsPerBlock network parameter.
+    int64 ms_per_block = 3 [json_name = "msPerBlock"];
```

Comment:
Maybe it would be better to have the Epoch duration in blocks here and get ms per block from chain parameters in NetworkConfig?

Reply:
I'm not sure what structure is better. I definitely want to separate the standard network parameters and the NeoFS-specific ones, and which of them will be in the `NetworkConfig` is under discussion. Current structure represents issue proposal. @alexvanin  what do u think?

Reply:
> I definitely want to separate the standard network parameters and the NeoFS-specific ones

Agree with that. As soon as the Epoch duration in blocks is a part of NeoFS Global configuration, I would not expect to see it in a separate field.

Reply:
I guess there is a bit of misunderstanding in the naming. `NetworkConfig` is a Global NeoFS Configuration, while other fields related to the chain (except current epoch).